nrpe/packaging/files/nrpe3_add_centreon_cmd.patch

```diff
@@ -0,0 +1,29 @@
+--- a/sample-config/nrpe.cfg.in	2018-09-14 13:40:08.496324915 +0200
++++ b/sample-config/nrpe.cfg.in	2018-09-14 13:47:57.624296953 +0200
```

Comment:
are these really intended?

Reply:
yes, this patch is executed properly before source compilation : 
![image](https://github.com/centreon/centreon-plugins/assets/11978823/b642b184-a670-4fef-b99d-6c3997994ec5)